package.json

```diff
@@ -70,7 +70,7 @@
     "typedoc-plugin-markdown": "^4.2.10",
     "typescript": "5.5.4",
     "typescript-eslint": "^7.18.0",
-    "stylelint": "^16.10.0",
+    "stylelint": "^16.12.0",
```

Comment:
Did you also update stylelint package from stylelint-config package.json file? We need to update on both places